src/pudl_rmi/coordinate.py

```diff
@@ -336,6 +411,120 @@ def check_is_file_or_not_exists(file_path: Path):
         )
 
 
+def pickle_plant_parts_distinct(plant_parts_eia, file_path):
+    """Pickle dataframe with only unique granularities from PPE."""
+    # Figure out where this should go. It's currently duplicated in
+    # connect_ferc1_to_eia.InputManager, should it be a PUDL function?
+    # can this be done more efficiently? use pipe?
```

Comment:
Left this as an RMI output for now but agree this should be a PUDL function eventually. Probably will pull this into PUDL when the CCAI entity matching stuff gets integrated with PUDL since the distinct PPL is necessary for that.